README.md

```diff
@@ -26,16 +26,16 @@ Build Gecko using the build script:
 ./scripts/build.sh
 ```
 
-The Gecko binary, named `ava`, is in the `build` directory.
+The Gecko binary, named `avalanche`, is in the `build` directory.
 
 ### Docker Install
 
 - Make sure you have docker installed on your machine (so commands like `docker run` etc. are available).
 - Build the docker image of latest gecko branch by `scripts/build_image.sh`.
 - Check the built image by `docker image ls`, you should see some image tagged
   `gecko-xxxxxxxx`, where `xxxxxxxx` is the commit id of the Gecko source it was built from.
-- Test Gecko by `docker run -ti -p 9650:9650 -p 9651:9651 gecko-xxxxxxxx /gecko/build/ava
-   --public-ip=127.0.0.1 --snow-sample-size=1 --snow-quorum-size=1 --staking-tls-enabled=false`. (For a production deployment,
+- Test Gecko by `docker run -ti -p 9650:9650 -p 9651:9651 gecko-xxxxxxxx /gecko/build/avalanche
```

Comment:
Would probably be better to give actual deployment instructions instead of for a one node network